paperoni/sources/scrapers/openreview.py

```diff
@@ -0,0 +1,228 @@
+from datetime import datetime
+from coleo import Option, tooled
+import openreview
+
+from ..model import (
+    Author,
+    DatePrecision,
+    Link,
+    Paper,
+    Release,
+    Topic,
+    Venue,
+    VenueType
+)
+from ...utils import QueryError
+
+class OpenReviewScraper:
+    @tooled
+    def query(
+        self,
+        # Author to query
+        # [alias: -a]
+        # [nargs: +]
+        author: Option = [],
+        # Title of the paper
+        # [alias: -t]
+        # [nargs: +]
+        title: Option = [],
+        # Maximal number of results per query
+        block_size: Option & int = 100,
```

Comment:
Does not appear to be used other than to override limit.

Reply:
True but that's enough to set the step size for each page. `limit` is still used to limit the total number of results but the `limit` in the openreview param dict is analog to `block_size`